knowage-vue/src/modules/qbe/qbeDialogs/qbeFilterDialog/QBEFilterValuesTable.vue

```diff
@@ -1,4 +1,5 @@
 <template>
+    {{ selectedValues }}
```

Comment:
I think you missed a print, can I remove it or you need it for debugging purpose?

Reply:
Yes, it can be removed. @Redjaw